core/client/fs/src/main/java/alluxio/client/file/FileSystemMasterClient.java

```diff
@@ -113,6 +113,13 @@ void createDirectory(AlluxioURI path, CreateDirectoryOptions options)
    */
   void free(AlluxioURI path, FreeOptions options) throws AlluxioStatusException;
 
+  /**
+   * Gets a map of metrics property names and their values from metrics system.
+   *
+   * @return a map of operation information
+   */
+  Map<String, Long> getMetrics() throws AlluxioStatusException;
```

Comment:
does this belong to fs master or meta master?
is fs master required here? I feel the metrics might be not related to fs metrics in the future.

Reply:
In the previous iteration, I only getOperation and getRpcInvocation, those are related to file system, but now we plan to get all the metrics data, put it in meta master is a better choice, I will move it to meta master! Thanks